draft-ietf-openpgp-pqc.md

```diff
@@ -310,7 +310,7 @@ This specification introduces new cryptographic schemes, which can be categorize
 
  - PQ digital signature, namely SLH-DSA-SHAKE as a standalone cryptographic algorithm.
 
-For each of the composite schemes, this specification mandates that the recipient has to successfully perform the cryptographic algorithms for each of the component schemes used in a cryptographic message, in order for the message to be deciphered and considered as valid.
```

Comment:
Since we're talking about signatures here, I guess "verifier" or something like that would work.

An alternative would be to simply state how the verification procedure works, not how an entity (recipient/comm. party) has to perform it. Something like "composite sigs are only considered valid if all component sigs are valid"

"communication party" might be weird in some contexts.

Reply:
I don't think it's verification only here, if we want to use `verifier` we also need to say `decrypter`?

Reply:
seems to me that we have a "generating" implementation and a "consuming" or "evaluating" implementation.  "Communication party" is rather too vague.